entrypoint

```diff
@@ -160,6 +160,16 @@ else
     # make the dump
     mysqldump -h $DBSERVER -u$DBUSER -p$DBPASS $DB_LIST | gzip > ${TMPDIR}/${TARGET}
 
+    # Execute additional scripts for post porcessing. For example, create a new
+    # backup file containing this db backup and a second tar file with the
+    # contents of a wordpress install.
+    if [ "$(ls -A /scripts.d/post-backup/)" ];
+    then
+      for i in $(ls /scripts.d/post-backup/*.sh);do
+          source $i
```

Comment:
I think it is overkill for this, doing the tasks I mentioned would be needed to be able to just make the scripts execute cleanly from the main script, also if you plan to see it as an API then you plan to use API versioning too to avoid breakage ? IMO documenting the available variables for use should be enough, but it's your code, finish it as you like.